File: app/src/main/java/io/palaima/calendar/MyYAxisValueFormatter.java
package io.palaima.calendar;


import java.text.DecimalFormat;

import io.palaima.chart.YAxis;
import io.palaima.chart.YAxisValueFormatter;

public class MyYAxisValueFormatter implements YAxisValueFormatter {

    private DecimalFormat mFormat;

    public MyYAxisValueFormatter() {
        mFormat = new DecimalFormat("###,###,###,##0.0");
    }

    @Override
    public String getFormattedValue(float value, YAxis yAxis) {
        return mFormat.format(value) + " $";
    }
}

File: app/src/main/java/io/palaima/calendar/MyYAxisValueFormatter.kt
package io.palaima.calendar


import io.palaima.chart.YAxis
import io.palaima.chart.YAxisValueFormatter
import java.text.DecimalFormat

class MyYAxisValueFormatter : YAxisValueFormatter {

    private val mFormat: DecimalFormat

    init {
        mFormat = DecimalFormat("###,###,###,##0.0")
    }

    override fun getFormattedValue(value: Float, yAxis: YAxis): String {
        return mFormat.format(value.toDouble()) + " $"
    }
}
